docker-compose.yml

```diff
@@ -28,5 +20,5 @@ services:
       - ./db/dbms:/data/dbms
     environment:
       - NEO4J_AUTH=neo4j/root
-      - NEO4J_dbms_memory_heap_initial__size=7G
-      - NEO4J_dbms_memory_heap_max__size=7G
+      - NEO4J_dbms_memory_heap_initial__size=1G
+      - NEO4J_dbms_memory_heap_max__size=1G
```

Comment:
my computer doesn't have 7G of ram ☹️

Reply:
okay for a very basic neo4j db it should be okay, writing could need a lot of memory.